src/deluge/memory/memory_region.h

```diff
@@ -38,7 +38,7 @@ struct NeighbouringMemoryGrabAttemptResult {
 #define SPACE_HEADER_ALLOCATED 0x80000000
 
 #define SPACE_TYPE_MASK 0xC0000000
-#define SPACE_SIZE_MASK 0x3FFFFFFF
+#define SPACE_SIZE_MASK 0x3FFFFFFFu
```

Comment:
Should add the `u` suffix to TYPE_MASK as well, as long as we're in the area

Reply:
Yeah sure, I left it since that one defaults to unsigned already